pkg/jx/cmd/install.go

```diff
@@ -117,6 +119,8 @@ type InstallFlags struct {
 	NoGitOpsVault               bool
 	NextGeneration              bool
 	StaticJenkins               bool
+	LongTermStorage             bool
```

Comment:
Are these options needed? I thought the intent was to  per default store into a GCS bucker in case the provider is GKE?

Reply:
Default answer will be yes, but can always give the option to disable.